### PR TITLE
fix(project-dashboard): solve project-dashboard QAs

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/project/components/ProjectDetailTabHeader.vue [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+const gnbStore = useGnbStore();
+
+const queryHelper = new QueryHelper();
+const storeState = reactive({
+    users: computed<UserReferenceMap>(() => allReferenceStore.getters.user),
+    projects: computed(() => allReferenceStore.getters.project),
+    costDataSource: computed<CostDataSourceReferenceMap>(() => allReferenceStore.getters.costDataSource),
+});
+const state = reactive({
+    menuItems: [
```
Comment (sulmoJ) [inline apps/web/src/services/project/components/ProjectDetailTabHeader.vue:96]: computed!

File: apps/web/src/services/project/components/ProjectDetailTabHeader.vue [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+
+
+/* Watchers */
+watch(() => projectDetailPageGetters.projectType, async () => {
+    await Promise.allSettled([
+        getProjectAlertActivated(),
+        listWebhooks(),
+        fetchUserList(),
+    ]);
+}, { immediate: false });
```
Comment (sulmoJ) [inline apps/web/src/services/project/components/ProjectDetailTabHeader.vue:334]: nitpick: If immediate is false, how about remove it.

File: apps/web/src/services/project/components/ProjectDetailTabHeader.vue [hunk 3]
```diff
@@ -0,0 +1,710 @@
...
+], ([id, globalGrantLoading]) => {
+    if (!globalGrantLoading) projectDetailPageStore.setProjectId(id);
+}, { immediate: true });
+
+watch(() => projectDetailPageState.projectId, (projectId) => {
+    gnbStore.setId(projectId);
+}, { immediate: true });
+// watch(() => state.favoriteOptions, (favoriteOptions) => {
+//     gnbStore.setFavoriteItemId(favoriteOptions);
+// }, { immediate: true });
```
Comment (sulmoJ) [inline apps/web/src/services/project/components/ProjectDetailTabHeader.vue:358]: This annotation is necessary??

**Review by sulmoJ — APPROVED** [review]
I check only convention~~
LGTM~

File: apps/web/src/services/project/components/ProjectMainProjectCard.vue [hunk 4]
```diff
@@ -180,13 +182,17 @@ const handleSelectItem = (selected: MenuItem) => {
             <div v-else
                  class="account-add"
             >
-                <router-link :to="{ name: ASSET_INVENTORY_ROUTE.SERVICE_ACCOUNT._NAME }">
+                <router-link :to="{ name: ASSET_INVENTORY_ROUTE.SERVICE_ACCOUNT._NAME }"
+                             @click.native.stop.prevent
```
Comment (yuda110) [inline apps/web/src/services/project/components/ProjectMainProjectCard.vue:186]: wow...

**Review by yuda110 — APPROVED** [review]
good!!!!!!

**Pull request merged**